deployment/ansible/roles/raster-foundry.docker/tasks/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- name: Uninstall docker-py
```

Comment:
I think that it is fine to assume that these provisioning steps are going to be applied against a clean machine. Doing stuff like this to account for previously installed versions is hard to get right and adds weird failure cases. I think we'd just ask for people to reprovision their VMs from scratch after this change goes in.